modules/file-system/parallelstore/README.md

```diff
@@ -1,3 +1,80 @@
+## Description
+
+This module creates [parallelstore](https://cloud.google.com/parallelstore)
+instance. Parallelstore is Google Cloud's first party parallel file system
+service based on [Intel DAOS](https://docs.daos.io/v2.2/)
+
+### Parallelstore Quota
+
+To get access to a private preview of Parallelstore APIs, your project needs to
+be allowlisted. To set this up, please work with your account representative.
+
+### Example - New VPC
+
+For parallelstore instance, Below snippet creates new VPC and configures private-service-access
+for this newly created network.
+
+The parallelstore instance created here can be used with Slurm cluster or compute
+VM running Ubuntu 22.04, debian 12 or HPC Rocky Linux 8.
+
+```yaml
+ - id: network
+    source: modules/network/vpc
+    settings:
+      mtu: 8896
+
+  - id: ps_connect
+    source: community/modules/network/private-service-access
+    use: [network]
+    settings:
+      prefix_length: 24
+
+  - id: ps_instance
+    source: modules/file-system/parallelstore
+    use: [network, ps_connect]
+```
+
+### Example - Existing VPC
+
+If you want to use existing network with private-service-access configured, you need
```

Comment:
Should we mention that the network must be configured to have private service connect?

Reply:
Yes, good point. Added details below.